uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/app.js

```diff
@@ -43,6 +43,7 @@ define([
     $log.debug('app.run');
 
     $rootScope.pathTpl = sharedSettings.pathTpl;
+    $rootScope.sharedSettings = sharedSettings;
```

Comment:
i'm not sure this is the correct approach, although i understand why you did this. If we are to take the component-based approach, then it would make sense to reduce the number of "global" dependencies in each component as much as possible. To do that, you can simply inject `shared-settings` in each component and then store the `sharedSettings.statusNames` in the scope